rwengine/src/engine/GameData.cpp

```diff
@@ -522,14 +522,19 @@ bool GameData::loadModel(ModelID model) {
     return true;
 }
 
-void GameData::loadIFP(const std::string& name) {
+void GameData::loadIFP(const std::string& name, bool cutsceneAnimation) {
     auto f = index.openFile(name);
 
     if (f.data) {
         LoaderIFP loader;
         if (loader.loadFromMemory(f.data.get())) {
```

Comment:
Let `LoaderIFP::loadFromMemory` accept `f.data` or `f`?

Reply:
In my opinion it's too much for this pull. (it changes design)